### PR TITLE
Issue 132 - Support for uploading Proguard mapping files

### PR DESCRIPTION
**Pull request by clhols**

### CODE REVIEW

**Comment by bobvanderlinden** [comment]
Thanks a lot. This works nicely. I've included it into my repository and packaged it using jitpack. For anyone else that wants to use this PR use the follwing in Gradle:

```
buildscript {
    ...
    repositories {
        ...
        maven { url "https://jitpack.io" }
    }
    dependencies {
        ...
        classpath 'com.github.bobvanderlinden:gradle-play-publisher:1.2.0-beta2'
    }
}
```

**Comment by theyonibomber** [comment]
Can this be merged please?

**Comment by ChristianBecker** [comment]
@clhols : Can you please rebase this to latest `master`?

File: src/main/groovy/de/triplet/gradle/play/PlayPublishApkTask.groovy [hunk 1]
```diff
@@ -54,6 +55,12 @@ class PlayPublishApkTask extends PlayPublishTask {
             }
         }
 
+        //Upload Proguard mapping.txt if available
+        if (variant.mappingFile != null && variant.mappingFile.exists()) {
+            AbstractInputStreamContent fileAsStream = new FileContent('application/octet-stream', variant.mappingFile)
```
Comment (ChristianBecker) [inline src/main/groovy/de/triplet/gradle/play/PlayPublishApkTask.groovy]: The type of `fileAsStream` can be inferred by using `def`

**Comment by clhols** [comment]
Maybe the if should be:
if (variant.mappingFile?.exists())

**Comment by ChristianBecker** [comment]
Yes, this would be more "Groovy-ish". If you change this, I'm happy to merge this!

**Comment by hardysim** [comment]
Great that this has been added now. Is there a release upcoming so we can all use it? :wink:

**Comment by ChristianBecker** [comment]
@hardysim: Maybe @bhurling can release a `1.2.0-beta2` in the next days. No promises!

**Comment by bhurling** [comment]
Available in `1.2.0-beta2`

**Pull request merged**